projects/module-1/colorandom.md

```diff
@@ -46,76 +46,73 @@ Then, as a team:
+![iteration 2](./assets/colorandom/iteration2.png)
 
-### Iteration 3 - Saving Palettes
+- When the page is initially loaded, all colors should have an unlocked icon in the bottom right corner
+- When a user clicks on the unlocked icon, a locked icon should appear. The locked/unlocked icons should toggle back and forth as the user clicks them.
+- When the `New Palette` button is clicked, only the unlocked colors should change. Locked colors should remain.
+- When the palette is updated, your Data Model should be updated too
 
-<img src="./assets/colorandom/saving-palette.png" alt="Saving palette">
+Note: [The locked and unlocked icons can be found here!](https://drive.google.com/drive/folders/1ZX31xVFdRAsLPzsL8G2_ujNQNXgFmKxD?usp=sharing)
 
-- When the user clicks Save Palette, the saved palette should show up in the saved palette side of the UI
-    - The palette here does not display hex codes, just blocks of the palette's color
-    - Each saved palette should be rendered with a button to delete the palette (you won't build out this functionality until a later iteration)
-- A new Palette should automatically be created and displayed in the main part of the app
+### Iteration 3 - Saving Palettes
 
-- _Note: Saved palettes do NOT need to persist on page load_
+![iteration 3](./assets/colorandom/iteration3.png)
 
+- Add a `Save Palette` button to match the design above
+- Add a `Saved Palettes` section to match the design above
+- When the page loads, there should be no saved palettes
+- When the user clicks `Save Palette`:
+  - the palette should be saved somewhere in your Data Model
+  - the palette should show up in the saved palette section to match the design above
+- A new Palette should automatically be created and displayed in the main part of the app
 
-### Iteration 4 - Locking colors
+Note: Saved palettes do NOT need to persist on page load
 
-<img src="./assets/colorandom/locking-color.png" alt="Locking a color">
+### Iteration 4 - Deleting Saved Palettes
 
-- A user should be able to click a color in the Palette and lock it
-    - The open lock icon/emoji should change to be a locked icon/emoji to indicate that the color is locked
+![iteration 4](./assets/colorandom/iteration4.png)
 
-<img src="./assets/colorandom/new-palette-locked.png" alt="Updating a palette with locked colors">
+- Add a delete button next to each saved palette to match the design above
+- When a saved palette's delete button is clicked:
+  - the palette is removed from your Data Model
+  - the palette is removed from the page
 
-- When the New Palette button is clicked, all the colors update as expected, except for any colors that are locked
-- A user should be able to unlock colors and continue updating the colors with the New Palette button as expected
+Note: [The delete icon can be found here!](https://drive.google.com/drive/folders/1ZX31xVFdRAsLPzsL8G2_ujNQNXgFmKxD?usp=sharing)
```

Comment:
Again, potentially we could use the `<section class="note">` for links to the resources.  Maybe the 2 links could be combined into one so that students aren't thinking these are 2 separate links?

Reply:
done!